### DISCUSSION TITLE
求助：无法正确的处理@tailwind指令

### DISCUSSION THREAD

**Discussion by Young0ne018** [NervJS/taro#14990] [Q & A]
我按照文档
https://docs.taro.zone/en/docs/tailwindcss/;
中的步骤使用tailwindcss，但是我发现构建的wxss产物中依然存在@tailwind的指令。似乎并没有正确的处理@tailwind指令；

这是demo的地址：https://github.com/Young0ne018/taro

postcss.config.js:
`module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
    'postcss-rem-to-responsive-pixel': {
      // 32 意味着 1rem = 32rpx
      rootValue: 32,
      // 默认所有属性都转化
      propList: ['*'],
      // 转化的单位,可以变成 px / rpx
      transformUnit: 'rpx',
    },
  },
}`

tailwind.config.js:
`module.exports = {
  content: ['./src/**/*.{html,js,ts,jsx,tsx,vue}'],
  corePlugins: {
    preflight: false
  },
  theme: {
    extend: {},
  },
  plugins: [],
}`

wxss:
![image](https://github.com/NervJS/taro/assets/82131809/0750d082-ab6d-48c1-93f3-97fdbfda5fc4)

**Comment by 86chenjie** [comment]
我也碰到了这个问题，同求解决。

**Comment by 86chenjie** [comment]
https://github.com/NervJS/taro/issues/14996 新建了个issue 欢迎讨论
